runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -643,6 +651,33 @@ public void onResponse(Call call, Response response) throws IOException
 		});
 	}
 
+	/**
+	 * Extracts the name of the player the user killed in a PvP environment and returns it as a string.
+	 * Specifically, this is necessary to add the player's name to the file name.
+	 *
+	 * @param chatMessage Message to parse for killed player's name
+	 * @return String This returns the killed player's name
+	 */
+	private String killedPlayerName(String chatMessage)
+	{
+		for (String killMessage:KILL_MESSAGES)
+		{
+			while (chatMessage.contains(killMessage))
+			{
+				chatMessage = chatMessage.replace(killMessage, "");
+			}
+		}
+		for (String killMessageTrim:KILL_MESSAGE_TRIM)
```

Comment:
this won't work if the player's name contains any of these strings

Reply:
Ah yeah, jumped the gun on this one thinking I had a solid solution. I'll give it another look over in the morning to fix that.